src/wms/src/main/java/org/geoserver/wms/capabilities/DimensionHelper.java

```diff
@@ -475,76 +475,89 @@ protected String getNumberRepresentation(
         final StringBuilder buff = new StringBuilder();
 
         if (DimensionPresentation.LIST == dimension.getPresentation()) {
-            for (Object val : values) {
-                if (val instanceof Number) {
-                    buff.append(val);
-                } else {
-                    NumberRange range = (NumberRange) val;
-                    buff.append(range.getMinimum())
-                            .append("/")
-                            .append(range.getMaximum())
-                            .append("/0");
+            if (dimension.getFixedValues() != null) {
```

Comment:
There is no need to redo the fixed values parsing logic here. It has been already done in the WMS class and this class is indeed using it to retrieve the value to encode.